test/gurka/test_matrix.cc

```diff
@@ -53,7 +53,7 @@ void check_matrix(const rapidjson::Document& result,
 }
 } // namespace
 
-class MatrixTest : public ::testing::Test {
+class MatrixTrafficTest : public ::testing::Test {
```

Comment:
also renamed the test, it's not anymore only traffic specific. the class here is though.